docs/learn/distributed-authority-quick-start.md

```diff
@@ -105,8 +105,9 @@ public class ConnectionManager : MonoBehaviour
            AuthenticationService.Instance.SwitchProfile(_profileName);
            await AuthenticationService.Instance.SignInAnonymouslyAsync();
 
-            var options = new CreateSessionOptions(_maxPlayers) {
-                Name = _sessionName
+            var options = new SessionOptions {
+                Name = _sessionName,
+                MaxPlayers = _maxPlayers
```

Comment:
Should this be this?
`var options = new SessionOptions() {
                Name = _sessionName,
                MaxPlayers = _maxPlayers
}..WithDistributedConnection();`